recipe/meta.yaml

```diff
@@ -1,12 +1,12 @@
-{% set version = "5.4.3" %}
+{% set version = "5.5.0" %}
 
 package:
   name: ipykernel
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/i/ipykernel/ipykernel-{{ version }}.tar.gz
-  sha256: 697103d218e9a8828025af7986e033c89e0b36e2b6eb84a5bda4739b9a27f3cb
+  sha256: 98321abefdf0505fb3dc7601f60fc4087364d394bd8fad53107eb1adee9ff475
 
 build:
   number: 1
```

Comment:
@conda-forge/bot, looks like the bot left this as `1` when updating the version

Reply:
nah, don't blame the bot, it was me: we merged #75 after the bot made this, and didn't give it enough time to re-run, most likely.